backend/src/handlers/video_handler.go

```diff
@@ -166,7 +166,7 @@ func (srv *Server) handleFavoriteVideo(w http.ResponseWriter, r *http.Request, l
 	if err != nil {
 		return newInvalidIdServiceError(err, "video_id")
 	}
-	isFavorited, err := srv.Db.FavoriteOpenContent(vidId, userID, video.OpenContentProviderID, nil)
+	isFavorited, err := srv.Db.FavoriteOpenContent(vidId, video.OpenContentProviderID, userID, nil)
```

Comment:
good catch, somehow this was flipped on the other links endpoint as well :thinking: